spidermon/contrib/actions/sentry/__init__.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import absolute_import
+from __future__ import print_function
```

Comment:
Just noticed now. As we are not using `print` in this action, this import can be removed without problems.

Reply:
Already did :)